lms/static/sass/course/_info.scss

```diff
@@ -48,7 +48,7 @@ div.info-wrapper {
     width: 100%;
     display: block;
 
-    h1 {
+    h1,h3 {
```

Comment:
It would be great to change this to use a class, rather than an element type. That would also make it clearer, because it seems strange to have ``h1`` and ``h3`` have the same rules, but then not include ``h2``.